config/default_records/pipeline_request_information_types/002_element_aviti_pipeline_request_information_types.wip.yml

```diff
@@ -1,12 +1,11 @@
----
-FlowcellRequestedInformationTypeForElementAviti:
-  pipeline_name: Element Aviti
-  request_information_type_key: requested_flowcell_type
-
 LowDiversityInformationTypeForElementAviti:
   pipeline_name: Element Aviti
   request_information_type_key: low_diversity
 
-PercentPhixRequestedInformationTypeForElementAviti:
+PercentPhixNeededInformationTypeForElementAviti:
```

Comment:
Looks like the 'requested' terminology is used in the `request_information_type` - name, key and label. Why the change to 'needed' in the key here?